src/Module/Search/Acl.php

```diff
@@ -0,0 +1,372 @@
+<?php
+
+namespace Friendica\Module\Search;
+
+use Friendica\BaseModule;
+use Friendica\Content\Widget;
+use Friendica\Core\Hook;
+use Friendica\Core\L10n;
+use Friendica\Core\Logger;
+use Friendica\Core\Protocol;
+use Friendica\Core\Search;
+use Friendica\Database\DBA;
+use Friendica\Model\Contact;
+use Friendica\Model\Item;
+use Friendica\Network\HTTPException;
+use Friendica\Util\Proxy as ProxyUtils;
+use Friendica\Util\Strings;
+
+/**
+ * ACL selector json backend
+ *
+ * @package Friendica\Module\Search
+ */
+class Acl extends BaseModule
+{
+	const TYPE_GLOBAL_CONTACT = 'x';
```

Comment:
Plz align the constants (beautification :) ). But I love the idea of using these constants!! Makes the code a lot more readable